doc/ref_arch/kubernetes/chapters/chapter04.rst

```diff
@@ -649,7 +679,7 @@ Architecture they must be implemented as per the following specifications:
    * - ra2.pkg.001
      - API-based package management
      - A package manager must use the Kubernetes APIs to manage application artifacts. Cluster-side components such as
-       Tiller are not supported.
+       Tiller must not be required.
```

Comment:
should it just be "not required"
```suggestion
       Tiller are not required.
```

Reply:
I would prefer to keep it like this, as "are not required" means that it is optional, be we would like to forbid its usage.